wee_slack.py

```diff
@@ -2587,8 +2602,33 @@ def subprocess_channel_leave(message_json, eventrouter, channel, team):
 
 
 def subprocess_message_replied(message_json, eventrouter, channel, team):
-    pass
-
+    if "message" not in message_json:
+        return
+    parent_ts = message_json["message"].get("thread_ts")
+    if parent_ts:
+        parent_message = channel.messages.get(SlackTS(parent_ts), None)
```

Comment:
`None` is unnecessary here, it defaults to that.

Reply:
✓